lib/index.js

```diff
@@ -193,7 +193,10 @@ internals.authenticate = async function(token, options, request, h) {
             credentials && typeof credentials === 'object'
               ? credentials
               : decoded,
-          artifacts: token,
+          artifacts: {
+            token,
+            decoded,
```

Comment:
we can probably include a `headers` field in here too, thoughts?

Reply:
We tend to only add features when people request them with a specific use case 
otherwise the code would get bloated quite fast. 💭